src/models/Bib.ts

```diff
@@ -44,10 +45,9 @@ export default class Bib {
     this.itemAggregations = result.itemAggregations || null
     this.hasItemDates = result.hasItemDates || false
     this.subjectHeadings = result.subjectHeadings || null
-    this.findingAid =
-      result.supplementaryContent?.find(
-        (el) => el.label?.toLocaleLowerCase() === "finding aid"
-      )?.url || null
+    this.findingAid = result.supplementaryContent
```

Comment:
the getfindingaid method checks for existence of result.supplementaryContent and returns null as a default, so I don't think you need this extra logic here.